app.js

```diff
@@ -1,88 +1,207 @@
-// Task 1 - Notes for JS Evaluation Predictions
-
-// List 3 expressions that surprised you or gave unexpected results, why?
-
-// List 2 exprsssions you felt comfortable interpreting at a glance.
-
-// List 1 expression you are still 'shaky' on, why?
-
-
-
-
+/* Task 1 - Notes for JS Evaluation Predictions */
+
+// List 1: expression you are still 'shaky' on, why?
+
+// List 2: expressions you felt comfortable interpreting at a glance.
+/* 4 + 4 / 2
+(2 + -7) * 3
+101 % 10
+12 - 4 % 3
+true && false
+true && !(false || false)
+!true && !(false || false)
+'cat' + 'dog'
+2 + 'pizza'
+5 >= 11
+5 === 5.0
+7 !== 7.1
+5 + 5 > 8
+6 + 6 != 12
+2 > 1 || false
+'true' === true
+10 % 2 === 0
+21 % 2 === 0
+21 % 2 !== 0
+21 % 2 === 1
+12 % 3 === 0
+9 % 3 === 0
+14 % 3 === 0
+'new york'[0]
+'new york'[1]
+'san francisco'[2 * 3]
+'engineering'[3].toUpperCase()
+'engineering'.indexOf('r') > -1
+'sciende'.indexOf('x') === -1
+
+
+// List 3: expressions that surprised you or gave unexpected results, why?
+2.5 * 'fish' // I never tried doing something like this. This one was surprising but expected
+'engineering'.indexOf('G') // I thought the result was undefined because we don't have an upper case G. I didn't know the result would be -1 when not existent
+'engineering'.indexOf('neer') // I thought the result would be undefined because it was given more than one character to return a index. 
+ */
 
 // Task 2 - Practice JS Basics
-// 1 - 
-
-// 2 - 
-
-// 3 - 
-
-// 4 - 
-
-// 5 - 
-
-// 6 - 
-
-// 7 - 
-
-// 8 - 
-
-// 9 -
-
-// 10 - 
-
-
-//// Task 3 - Conditionals Practice 
-// 1 - 
-
-// 2 - 
-
-// 3 - 
-
-// 4 - 
-
-// 5 - 
-
-// 6 - 
-
-// 7 - 
-
-// 8 - 
-
-// 9 -
-
-// 10 - 
-
-//// Task 4 - Conditionals Muscle Memory
-// 1 - 
-
-// 2 - 
-
-// 3 - 
-
-// 4 - 
-
-// 5 - 
-
-// 6 - 
-
-// 7 - 
-
-// 8 - 
+// 1 -
+let idx = 'abcde'.indexOf('D')
+idx = idx + 11
+console.log(idx) // 10
+idx * 2
+console.log(idx) // 20 - wrong! I didn't pay attention here and thought the calculation on line 49 would automatically store the new value.
+
+// 2 -
+let num = 33
+let isEven = num % 2 === 0
+console.log(isEven) // false
+console.log(!isEven) // true
+
+// 3 -
+let str1 = 'marker'
+let num2 = 'whiteboard'.length - str1.length
+console.log(num2) // 4
+let str2 = 'bootcamp'
+console.log(str2[num2].toUpperCase()) // C
+let char = str2[num2].toLowerCase() // c
+console.log(char + '!') // c!
+
+// 4 -
+let sentence = 'welcome to bootcamp prep'
+let lastChar = sentence[sentence.length - 1]
+console.log(lastChar) // p
+console.log(sentence.indexOf(lastChar)) // 18
+
+//Task 3 - Conditionals Practice
```

Comment:
What were your predictions for task 3?

Reply:
Predictions were just added to the code :)